datasets/covid_tweets_japanese/README.md

```diff
@@ -0,0 +1,104 @@
+---
+YAML tags:
+- {"task_categories": ["classification"], "task_ids": [], "multilinguality": ["monolingual"], "languages": ["ja"], "language_creators": [], "annotations_creators": [], "source_datasets": ["original"], "size_categories": ["53,640 tweets"], "licenses": ["CC-BY-ND 4.0"]}
```

Comment:
It looks like you copy paster the JSON tags here instead of YAML
Could you use the yaml tags instead ? (even though the tagger allows to save in json)
You can find the yaml at the top right of the tagger by clicking `expand yaml tags`

Reply:
Thank you for pointing out that! I'll soon fix it.

Reply:
Updated README.md with help from https://raw.githubusercontent.com/huggingface/datasets/master/datasets/capes/README.md.

I can't find the information about task_id and some other items. 
I hope to add the information by working on fixing the dummy_data part and adding other datasets, gaining a better understanding of the repository.